.devcontainer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/go/.devcontainer/base.Dockerfile
+
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.18, 1.17, 1-bullseye, 1.18-bullseye, 1.17-bullseye, 1-buster, 1.18-buster, 1.17-buster
+ARG VARIANT="1.18-bullseye"
```

Comment:
I think `1.17` would be the safer option until we update our CI scripts and CONTRIBUTING instructions

Reply:
@natemoo-re Thanks for the review. Appreciate it. I'm pushing it now. 

FWIW, in the future, I'd like to engineer something a little cleaner. Just sticking to the VSCode recommendations for now though.